### ISSUE TITLE
missing_doc false negative

### ISSUE THREAD

**Issue by andrew8712** [realm/SwiftLint#759]
I have an open-source project migrated to Swift 3. Now I'm trying to add a documentation to it. However, `SwiftLint` is complaning about `Missing Doc Violation: Public declarations should be documented`, even for added ones:

```
/// A base button class.
@IBDesignable public class DCBaseButton: UIButton {
    ....
}
```
The libarary is here:

https://github.com/andrew8712/DCKit/tree/documentation

May be Swift 3 causing this?

**Comment by andrew8712** [comment]
A duplicate of https://github.com/realm/SwiftLint/issues/728
